pywb/warcserver/http.py

```diff
@@ -1,27 +1,66 @@
-from requests.adapters import HTTPAdapter
-import requests
 import os
 
+import requests
 import six.moves.http_client
+from requests.adapters import DEFAULT_POOLBLOCK, HTTPAdapter
+from urllib3.poolmanager import PoolManager
+
 six.moves.http_client._MAXHEADERS = 10000
+six.moves.http_client._MAXLINE = 131072
 
 SOCKS_PROXIES = None
 orig_getaddrinfo = None
 
 
-#=============================================================================
+def should_use_custom_adaptor():
```

Comment:
To simplify, think we should just use the pywb adapter always, no need to check version.
The adapter init is simply enough and backwards compatible

Reply:
done